rfp-proposals/rfp-community-proposal-GrandhelmsmanDocumentary.md

```diff
@@ -0,0 +1,93 @@
+# RFP Community Proposal: `IPFS documentary of Chinese college students `
+
+**Link to RFP:** Please link to the RFP that you are submitting a proposal for.
+
+**RFP Category:** `community`
+
+**Proposer:** `IPFS-grandhelmaman`
+
+# Event Details
+
+**Name of Event:** IPFS documentary of Chinese college students
+
+**Holding Country:** China
+
+**Location:** Chinese colleges and universities
+
+**Event Date:** In the fourth quarter of 2019
+
+**Event Time:** Include time zone
+
+**Event Link:** 
+
+**Expected Attendance:** 
+
+**Event Website:** 
+
+## Event Description
+
+Please describe in detail what type of event you are planning to host. Please include:
+- Objective of this Activity: to record the exploration of IPFS technology among Chinese college students.
+- The key point of the plan: interviewing and shooting.
+- Whether to invite speakers:
+- Photography and audio recording program: yes, It will be presented as a documentary.
+- Food and beverage plan: not yet.
+- Marketing and promoting party planning:
+- Plans to share records and post-event information:
+- Any follow-up activities you plan to organize:
+
+## Value
+
+What is the value of this event for the Filecoin ecosystem and community?
+
+Through this event, IPFS technology is gaining popularity among Chinese students, Guiding students to explore IPFS technology independently. In addition, the most innovative students can realize their inspiration by creating IPFS technology application.
```

Comment:
Thanks for the proposal! Unfortunately, we're only looking for proposals for Filecoin right now, not IPFS. You could reach out to the IPFS team in order to share this proposal with them!